content/docs/getting-started/integrations/gitlab.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: Integrating Kraft in GitLab CI
+description: |
+  Translating the CI system of GitHub to GitLab to allow Unikraft to run.
+---
+
+## Integrating Kraft in GitLab CI
+
+Kraftkit currently offers good integration with GitHub Worflows, as it offers even a [GitHub Action](https://github.com/marketplace/actions/build-unikernel-images-with-unikraft) that eases the call to `kraft` and takes advantage of the GitHub ecosystem.
+Because of this, GitLab has been left partially neglected, as it uses a separate format from GitHub to describe its automation, even though it is still `YaML`.
```

Comment:
CAn you remove text about GitHub, this integration page should focus only on GitLab.